### PR TITLE
OSX support and better documentation

### PR DESCRIPTION
**Pull request by jbmouret**

### CODE REVIEW

File: scripts/install_deps.sh [hunk 1]
```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
```
Comment (costashatz) [inline scripts/install_deps.sh]: @jbmouret Most probably you forgot to `OS=$(uname)`

File: scripts/install_deps_req.sh [hunk 2]
```diff
@@ -1,21 +1,34 @@
...
 mkdir -p install
 
-# do libgtest fix for libcmaes
-cd /usr/src/gtest
-sudo mkdir -p build && cd build
-sudo cmake ..
-sudo make
-sudo cp *.a /usr/lib
+# do libgtest fix for libcmaes (Linux only)
+if [ $OS = "Darwin" ]; then
```
Comment (costashatz) [inline scripts/install_deps_req.sh]: If the comment is correct, this should be `if ! [ $OS = "Darwin" ]; then` right?

**Comment by costashatz** [comment]
@jbmouret Are you sure that the new sdl detection works for linux too?

**Comment by costashatz** [comment]
> @jbmouret Are you sure that the new sdl detection works for linux too?

I checked this and it works on linux too.

We should also make the other scripts work too. Or we should add some instructions on how to compile without the scripts..

**Comment by jbmouret** [comment]
I think we do not need Dart for OSX. I am fine with advertising with the current version (if this still works on Linux).

**Comment by costashatz** [comment]
> I think we do not need Dart for OSX. I am fine with advertising with the current version (if this still works on Linux).

OK (it works on linux). I will just add some instructions (very quick) on how to install/configure/compile everything without the scripts and then we should advertise it as is.

**Comment by costashatz** [comment]
@jbmouret I think this PR is ready. Please do a quick check and then we should merge!

**Comment by costashatz** [comment]
**Merging...**

**Pull request merged**